modules/javafx.controls/src/main/java/javafx/scene/control/TreeTableView.java

```diff
@@ -1804,6 +1804,11 @@ public int getVisibleLeafIndex(TreeTableColumn<S,?> column) {
         return visibleLeafColumns.get(column);
     }
 
+    boolean sortingInProgress;
+    protected boolean isSortingInProgress() {
```

Comment:
This is an implementation detail. By making it `protected` it becomes public API. You will need to make it package-scope instead

Reply:
Updated in next commit